_sass/components/_c-section-title.scss

```diff
@@ -7,6 +7,7 @@
   line-height: 1.55em;
   text-align: center;
   font-family: $font-family-primary;
+  text-transform: capitalize;
```

Comment:
I prefer to add the C directly to the title. 
If you use capitalize, it will capitalize the first letter to very word in the title. And we dont want that.